lib/inertia_phoenix/plug.ex

```diff
@@ -49,4 +50,17 @@ defmodule InertiaPhoenix.Plug do
     |> put_status(:conflict)
     |> halt()
   end
+
+  def check_redirect(conn) do
+    conn
+    |> Plug.Conn.register_before_send(fn conn ->
+      case(conn.method in ["PUT", "PATCH", "DELETE"] and conn.status in [301, 302]) do
+        true ->
+          conn |> put_status(303)
```

Comment:
This could be better written without pipes, I think it's even suggested by Elixir (or at least Credo) to not do "single piping"